### DISCUSSION TITLE
stopped at starting dendron

### DISCUSSION THREAD

**Discussion by realjustfree** [dendronhq/dendron#2022] [Q&A]
thanks for the development of dendron

recently dendron can't be loaded on vscode
it stayed on starting dendron stage on vscode excuted

the troubleshooting page on 'dendron is stuck on loading' only says that fixation is Whitelisting localhost 
but IMO my localhost setting is normal.
I just think that recent update of dendron cause this problem.

![image](https://user-images.githubusercontent.com/26951235/147841734-67088b57-7824-4fcc-ae1f-bb398fc0d5ea.png)

is there other fixation for this? 
thanks for reading,

**Comment by hikchoi** [comment]
Hey @realjustfree, I can't seem to reproduce this on version 0.75.1

Can you run `Dendron:Dev Open Logs` in the command palette and share the log in there?
If you aren't comfortable sharing this in the public, you can join our [Discord server](https://discord.gg/jqsCqZsy) and message me directly (I am hikchoi#8934 in Discord).

**Comment by hikchoi** [comment]
Had a private conversation with @realjustfree, and leaving this information here for visibility:

Due to various restrictions on certain networks, some machines need capability to configure their local server host. We will introduce this week so that users aren't blocked from using Dendron because of network restrictions.
